src/components/pages/species/list/header/index.tsx

```diff
@@ -0,0 +1,13 @@
+import { PageHeading } from "@components/@core/layout";
+import useTranslation from "@hooks/use-translation";
+import React from "react";
+
+export default function Header() {
```

Comment:
if this component is not shared ,could pls use it directly in its desired component itself

Reply:
@teamdraftbox done!